recipes/isim/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "isim" %}
+{% set version = "15.0.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mqcomplab/iSIM/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 306dc02bc9a57772b49c3a4c96010e567b9cdf92a818d2e8f362e0cea7dcb949
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
```

Comment:
`host` should likely only need to include `python`, `pip` and `setuptools`. I'm not seeing any dependency on `poetry` in the code in the tarball

Reply:
Let's try removing everything after `setuptools` here under `host` and assuming that passes, we should be ready to merge.